src/System.Numerics.Vectors/tests/Vector2Tests.cs

```diff
@@ -29,7 +28,6 @@ public void Vector2CopyToTest()
             Assert.Throws<NullReferenceException>(() => v1.CopyTo(null, 0));
             Assert.Throws<ArgumentOutOfRangeException>(() => v1.CopyTo(a, -1));
             Assert.Throws<ArgumentOutOfRangeException>(() => v1.CopyTo(a, a.Length));
-            Assert.Throws<ArgumentException>(() => v1.CopyTo(a, 2));
```

Comment:
This line shouldn't be deleted - as a minbar, it should be guarded like this:

```
      if (!PlatformDetection.IsNetNative)
           Assert.Throws<ArgumentException>(() => v1.CopyTo(a, 2));
      else
           Assert.ThrowsAny<ArgumentException>(() => v1.CopyTo(a, 2));  // .Net Native behavior: Explanation
```

I'll leave it to you to come up with the one-line "Explanation" part.